modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -48,14 +48,12 @@ public function buildForm(array $form, FormStateInterface $form_state, array $co
     $form_state->set('oe_whitelabel_search_config', $config);
     $input_value = '';
 
-    $form['#attributes']['class'][] = 'bcl_search_form';
-
     if (!empty($config['input']['name'])) {
       $input_value = $this->getRequest()->get($config['input']['name']);
     }
 
     $form['search_input'] = [
-      '#prefix' => '<div class="bcl-search-form__group">',
+      '#prefix' => '<div class="bcl-search-form__group w-100">',
```

Comment:
I think `flex-grow-1` would be more logical here. But `w-100` does the job, so let's keep it.